assets/prefabs/recipe/cook/RecipeToast.prefab

```diff
@@ -0,0 +1,19 @@
+{
+    "ProcessDefinition": {
+        "processType": "Cooking:Cooking"
+    },
+    "CraftingStationRecipe": {
+        "recipeId": "Hot food|Cooking:Toast",
+
+        "recipeComponents": ["1*Bread"],
+        "recipeFluids": ["0.25*Fluid:Water"],

```

Comment:
Sorry, I was going to remove it but then I looked at all the other prefabs and ALL of them (unless I missed some) seemed to have at least some amount of water, even though it might not make sense so I thought it was a requirement, I would probably prefer to have no water

I wanted to make it a cooking recipe because toast obviously requires a toaster... I could look into possibly making a utensil called toaster but its sole purpose is to just make bread into toast so I was still kinda iffy on that